### PR TITLE
Prepare new release 1.1.0

### PR DESCRIPTION
**Pull request by gwarf**
Prepare new release 1.1.0

```
## [1.1.0] - 2023-03-27

### Added

- Import community files and GitHub actions. Target CentOS 7, AlmaLinux 8 and 9. (#2) (Baptiste Grenier)
```

### CODE REVIEW

**Comment by gwarf** [comment]
@andrea-manzi , @enolfc this one is ready, I tested the release in my fork it's now working: https://github.com/gwarf/bdii-config-top/actions/runs/4541462542 https://github.com/gwarf/bdii-config-top/releases/tag/v1.1.0
For the record as my fork was having only Read permissions for Workflow actions, I had to look into setting [custom permissions](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token), allowing to have them explicitly documented in the yaml file, and avoiding to have to set write permissions for all scopes.

File: tests/test-top-bdii [hunk 1]
```diff
@@ -21,63 +21,59 @@ sleep 400
...
+fi
 
 echo -n "Checking resource BDII publication ... "
 bind="mds-vo-name=resource,o=grid"
 filter="(&(objectClass=GlueService)(GlueServiceType=bdii_top))"
-${command} ${bind} '${filter}' >/dev/null 2>/dev/null
-if [ $? -gt 0 ]; then
-    echo "FAIL"
-    RETVAL=1
+if ! ${command} ${bind} "${filter}" >/dev/null 2>/dev/null; then
```
Comment (andrea-manzi) [inline tests/test-top-bdii:33]: shouldn't this be if [ ! ${command} ...]?

Comment (gwarf) [reply tests/test-top-bdii:33]: no :)

Comment (gwarf) [reply tests/test-top-bdii:33]: See the POSIX definition of `if`: 
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_04_07.

The `[]` are basically a shortcut for using the `test` command, see https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html#tag_20_128

Comment (gwarf) [reply tests/test-top-bdii:33]: And the usage of `!` is here: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_02

Comment (andrea-manzi) [reply tests/test-top-bdii:33]: ok thanks! i understand now how it works:-)

**Pull request merged**